lib/modules/modules.go

```diff
@@ -331,7 +332,7 @@ func GetModules() Modules {
 var ErrCannotDisableSecondFactor = errors.New("cannot disable multi-factor authentication")
 
 // ValidateResource performs additional resource checks.
-func ValidateResource(res types.Resource) error {
+func ValidateResource(res any) error {
```

Comment:
Is there any reason that `AutoUpdateVersion` can't implement `types.Resource`?

Reply:
new resources implements `types.Resource153` they not compatible with each other

Reply:
agree, changing this one